packages/iso639/iso639.0.0.4/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "iso639"
+maintainer: "Petter A. Urkedal <paurkedal@gmail.com>"
+authors: "Petter A. Urkedal <paurkedal@gmail.com>"
+license: "LGPL-3 with OCaml linking exception"
+homepage: "https://github.com/paurkedal/ocaml-iso639/"
+doc: "http://paurkedal.github.io/ocaml-iso639/index.html"
+bug-reports: "https://github.com/paurkedal/ocaml-iso639/issues"
+depends: [
+  "ocaml" {>= "4.04.2"}
+  "dune" {build}
+  "ocamlfind" {build}
```

Comment:
```suggestion
```
you don't seem to use ocamlfind anywhere in this package

Reply:
I'll remove it. Thanks!